qiskit_ibm_runtime/api/rest/cloud_backend.py

```diff
@@ -15,7 +15,7 @@
 from typing import Dict, Any, Optional
 from datetime import datetime as python_datetime
 
-from qiskit_ibm_provider.api.rest.base import RestAdapterBase
+from qiskit_ibm_runtime.api.rest.base import RestAdapterBase
```

Comment:
I see differences between these classes, is it ok if we just use the existing one in this repo instead of the other from `qiskit-ibm_provider` ?

Reply:
Yeah the version in `qiskt-ibm-provider` has the method `get_facade_url` which is no longer necessary, so it's ok to use the version in this repo